### DISCUSSION TITLE
Split Changelog into per-version .md file

### DISCUSSION THREAD

**Discussion by lunaleaps** [reactwg/react-native-releases#9] [Announcements]
Problem:
 * It's currently very hard to navigate the changelog as the file is super long and VSCode (with VIM extensions) lags when moving around 😢
* Hard to `cmd+f` things in certain sections because you might edit the wrong version etc.  

Proposal:
* Let's split up the changelog.md file into separate files
* Let's create an artifact for the `rn-changelog-generator` script to consume (which will just be the shas) to use for filtering but generating as separate file for every release from now on.. or even retro-actively splitting everything

**Comment by Simek** [comment]
I'm against splitting the changelog to the separate files, because I think that this will increase the confusion with time and it would be hard to find specific entry or change, when you don't know when a feature has been exactly added or a bug has been fixed (in which version). 

The short and divided to the versions changelog can be listed on the GH Release pages (and that's what some of the OSS project do), to avoid the "big file problem" we can link GH releases instead of the Changelog file in community communication, and provide the link to the full changelog in the releases post. Putting those additional data in the release post should be quite strait-forward, since final changelog is already formatted for MD display.

* https://github.com/facebook/react-native/releases/tag/v0.67.0

**Reply by kelset** [reply]
yeah I agree with Simek, I'd prefer just 1 changelog.md file

**Reply by lunaleaps** [reply]
I think we can do both. I think my main gripe with the large changelog file is that whenever we need to manually edit -- which we do in the PR for the changelog for the release candidate -- it's really cumbersome and hard to know what version you're editing. I've def. accidentally edited the wrong version before. 

What if we had separate version changelogs that we concated into one changelog.md just for readable format -- but it has no direct edits? Alternatively, if we improve the changelog generator (re: a lot of tasks I listed in release improvements) we wouldn't have to manually edit it as frequently. but alas.

**Comment by cortinico** [comment]
Was there any agreement on this? I noticed that this became stale: https://github.com/facebook/react-native/issues/32935

If we're not looking into doing this, let's close the issue/discussion? 👍

**Reply by kelset** [reply]
I'd close both
